gatsby/src/pages/index.js

```diff
@@ -10,7 +10,7 @@ import Hero from '../components/HomeHero';
 import GetInvolved from '../components/HomeGetInvolvedSection';
 import RelatedMeetups from '../components/HomeRelatedMeetupsSection';
 
-export const HomeStyles = styled.div`
+const HomeStyles = styled.div`
```

Comment:
This export was throwing a build warning.

Reply:
Oh yeah that was totally not right, nice catch.